test/orderby.js

```diff
@@ -217,3 +217,139 @@ test('/team?$orderby=includes__pilot/$count($filter=is_experienced eq true) desc
 			]);
 	});
 });
+
+test(`/pilot?$orderby=identification_method(1)/identification_number desc`, (result) => {
+	it('should order pilots by their passport number when using a promitive key', () => {
+		expect(result)
+			.to.be.a.query.that.selects(pilotFields)
+			.from('pilot', ['identification method', 'pilot.identification method'])
+			.where([
+				'And',
+				[
+					'Equals',
+					['ReferencedField', 'pilot', 'id'],
+					['ReferencedField', 'pilot.identification method', 'pilot'],
+				],
+				[
+					'IsNotDistinctFrom',
+					['ReferencedField', 'pilot.identification method', 'id'],
+					['Bind', 0],
+				],
+			])
+			.orderby([
+				'DESC',
+				[
+					'ReferencedField',
+					'pilot.identification method',
+					'identification number',
+				],
+			]);
+	});
+});
+
+test(`/pilot?$orderby=identification_method(pilot=1,identification_type='passport')/identification_number desc`, (result) => {
```

Comment:
The `pilot=1` here would break things right? Since it would give the same results as doing `/pilot(1)` which I doubt is expected/intended

Reply:
If by breaking you mean that it will remove all other `pilot` rows, then yes...
After #160 gets merged though, this would query will still return all rows and just order by whatever is passed in here.
Do you think we should make this throw an not allow the navigated FK to be used in the alternate key regardless of #160 ?

Reply:
I think it would make sense for this to be blocked if we are not allowing arbitrary properties to be added for filtering purposes (since in this case specifying the pilot is only adding an *additional* filter on top of the necessary navigation part)